### DISCUSSION TITLE
How to add a css class used in HTMLString?

### DISCUSSION THREAD

**Discussion by vv-ac** [filamentphp/filament#2234] [Help]
Hi all,

I use the following code to color a Placeholder field:
```
->extraAttributes(function($record) {
                                                        if($record->type === 'important') {
                                                            return ['class' => 'bg-cyan-600'];
                                                        }
                                                        else {
                                                            return ['class' => 'bg-violet-600'];
                                                        }
                                                        })
```

When running npm run watch, it compiles successfully but it seems like it's missing out css classes that are used in this file.

I listed the folder in tailwind.config.js
```
module.exports = {
  content: [
  './storage/framework/views/*.php',
  './resources/**/*.blade.php',
  './resources/**/*.js',
  './resources/**/*.vue',
  "./vendor/laravel/framework/src/Illuminate/Pagination/resources/views/*.blade.php",
  "./app/Filament/*.php",
],
````

Am I missing out on sth here?

Thank you!

**Comment by vv-ac** [comment]
Oh, right I also added 
```
        <link href="/css/app.css" rel="stylesheet">
```
in my base.blade.php

**Reply by vv-ac** [reply]
Just re-read my own post here. Adding the css did not do the trick to clarify that. I am still looking for a way to have those classes be part of my app.css eventually.

**Reply by danharrin** [reply]
You'll need to follow the instructions to build your own theme - https://filamentphp.com/docs/2.x/admin/appearance#building-themes - but also adding your app/Filament PHP files to `tailwind.config.js`.
